cmd/ls.go

```diff
@@ -42,14 +42,14 @@ func printFolderMetadata(w io.Writer, e *files.FolderMetadata, longFormat bool)
 	if longFormat {
 		fmt.Fprintf(w, "-\t-\t-\t")
 	}
-	fmt.Fprintf(w, "%s\n", e.Name)
+	fmt.Fprintf(w, "%s\t\t", e.Name)
```

Comment:
This won't work well when you have a lot of files (e.g. lines won't wrap nicely)